cmake/protobuf-config.cmake.in

```diff
@@ -1,11 +1,17 @@
 # Version info variables
 set(PROTOBUF_VERSION        "@protobuf_VERSION@")
 set(PROTOBUF_VERSION_STRING "@protobuf_VERSION_STRING@")
+set(PROTOBUF_USE_ZLIB       "@protobuf_WITH_ZLIB@")
+
 
 # Current dir
 get_filename_component(_PROTOBUF_PACKAGE_PREFIX
   "${CMAKE_CURRENT_LIST_FILE}" PATH)
 
+if (PROTOBUF_USE_ZLIB)
+  find_package(ZLIB)
```

Comment:
I can remove it. But in that case you have to add explicitly find_package(ZLIB) to the CMakeLists.txt file of your project otherwise you have error message at target_link_libraries() command, since ZLIB is used by default. I thought it is better solution to be independent from the compile parameter of a thirdpary library.